### DISCUSSION TITLE
1password - local desktop app not connecting

### DISCUSSION THREAD

**Discussion by harmgenius** [Alex313031/thorium#412] [Q&A]
Super impressed with this great Thorium browser!

Question; the 1password extension connects to local desktop app (usually) but in Thorium it doesn't??

Is there a setting that prevents this?

See service worker output:
background.js:2 👋 Initializing 1Password
background.js:2 channel: nightly
version: 2.17.24
build: 21702400 (117.0.5938.157)
browser: Chrome (117.0.5938.157)
os: Windows (11.0)
background.js:2 WASM: Initializing. Log Level: Info
background.js:2 Attempted to update BugSnag config, but config has not yet been set.
Ms @ background.js:2
background.js:2 💫 Looking for desktop app com.1password.1password
background.js:2 📤 Sending message to native core <419806158>
background.js:2 👍 Finished initializing 1Password
chrome-extension://g…und/background.js:1 Uncaught (in promise) Error: Could not establish connection. Receiving end does not exist.
background.js:2 Received from the native core
2chrome-extension://g…und/background.js:1 Uncaught (in promise) Error: Could not establish connection. Receiving end does not exist.
background.js:2 ⏳ Timeout waiting for response to from native core <419806158>

**Comment by gz83** [comment]
There are multiple similar or identical issues in the repo related to Thorium

**Comment by harmgenius** [comment]
Sorry – missed it!

Have raised it again, so sorry for duplication.
